pyquil/api/_compiler.py

```diff
@@ -425,6 +431,20 @@ def calibration_program(self) -> Program:
         else:
             return self._calibration_program
 
+    def expand_calibrations(self, program: Program, discard_defcals: bool = True) -> Program:
```

Comment:
IMO this logic should be on `Program`, or in a standalone function, for use where a `QPUCompiler` doesn't make sense